src/modules/fundingManager/IFundingManager_v1.sol

```diff
@@ -37,6 +37,10 @@ interface IFundingManager_v1 {
     /// @param _amount The amount of underlying tokens transfered.
     event TransferOrchestratorToken(address indexed _to, uint _amount);
 
+    /// @notice Event emitted when collateral token has been set
+    /// @param token The token that serves as collateral token making up the curve's reserve
```

Comment:
Not every fundingManager is made up of a curve. With that the comment here is incorrect

Reply:
In general I dont know if I would use the word collateral here.
"Collateral is an item of value pledged to secure a loan." Is the first sentence on google that pops up for the word and I dont think thats the case for a rebasing or Deposivault FundingManager.
Maybe OrchestratorToken? (Dont know about that one though)